scripts/npm-link.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -e
+
+if [ "$CI" = "true" ]; then
```

Comment:
If you don't care what CI is set to (just that it isn't empty), you could use `-z`:

```bash
if [ ! -z "$CI" ]; then
```